cves/CVE-2013-0870.yml

```diff
@@ -64,7 +64,13 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
```

Comment:
This description is a bit too vague. What are the security implications?

Reply:
I agree with that, you kind of lost me starting in the "Basically, ..." sentence